engine/sound/src/wscript

```diff
@@ -28,21 +28,15 @@ def build(bld):
         source += ['sound_generic.cpp']
 
     if 'android' in bld.env.PLATFORM:
-        #include = 'openal/include'
         source += ['devices/device_opensl.cpp']
-    elif 'web' in bld.env.PLATFORM:
-        source += ['devices/device_js.cpp']
     elif 'nx64' in bld.env.PLATFORM:
         source += ['devices/device_nx64.cpp']
     elif bld.env.PLATFORM in ('x86_64-ps4','x86_64-ps5',):
         source += ['devices/device_ps4.cpp']
     else:
         source += ['devices/device_openal.cpp']
-        bld.recurse('openal')
```

Comment:
The folder was only used on Android, but we already use another code path for Android.